product_pricelist_supplierinfo/models/product_product.py

```diff
@@ -24,3 +24,35 @@ def price_compute(self, price_type, uom=False, currency=False,
             return dict.fromkeys(self.ids, 1.0)
         return super().price_compute(
             price_type, uom=uom, currency=currency, company=company)
+
+    @api.multi
+    def _select_seller_without_min_qty(self, partner_id=False, date=None, params=False):
+        """Method base on the original Odoo's method '_select_seller' but in here the
+        validation of the min_qty on the seller is being avoided with the use of the
+        field `no_supplierinfo_min_quantity` on the model `product.pricelist.item`.
+        Odoo's _select_seller() method:
+        https://github.com/odoo/odoo/blob/d17c20e28879d4abd951d56f367e4c1fd21df/addons/product/models/product.py#L566
+        """
+        self.ensure_one()
+        if date is None:
+            date = fields.Date.context_today(self)
+
+        res = self.env['product.supplierinfo']
+        sellers = self._prepare_sellers(params)
+        company_id = self.env.context.get('force_company')
+        if company_id:
+            sellers = sellers.filtered(
+                lambda s: not s.company_id or s.company_id.id == company_id)
+        for seller in sellers:
+            if seller.date_start and seller.date_start > date:
+                continue
+            if seller.date_end and seller.date_end < date:
+                continue
+            if partner_id and seller.name not in [partner_id, partner_id.parent_id]:
```

Comment:
Consider using `commercial_partner_id` instead of `parent_id`

Reply:
As I'm using the original code from the `_select_seller` should I change it?

Reply:
I just checked the original code. Even though you would be changing it, I think it's kind of clear you would be doing the same thing.

Reply:
Sorry, I replied to the wrong comment. I think the original one should use the commercial partner. But, since it does not let's stick with that way.

Reply:
Ok, done